docs/user-guide/documents/upload-documents.md

```diff
@@ -52,6 +52,16 @@ You will need to [Create a project](/user-guide/projects/create-project) and [Cr
     - `Merge` option - If unchecked, uploaded translations overrides current translation, otherwise, it will merge with current translation in system.
     - `My translations` option - Indicates if all uploaded translations were translated by you.
 
+#### Translation document formats
+Translation files, typically of the Gettext .po format, can be uploaded for a document in the selected language.
+There is a limited set of document formats that support upload of a translated raw document. These are:
+ * Properties (.properties)
```

Comment:
Does this really work though? You can't currently upload .properties to the FileService, nor to the web UI, AFAIK.

Reply:
Done (removed)